corehq/apps/app_manager/tests/test_suite.py

```diff
@@ -393,7 +392,28 @@ def test_graphing(self):
         self._test_generic_suite('app_graphing', 'suite-graphing')
 
     def test_fixtures_in_graph(self):
-        self._test_generic_suite('app_fixture_graphing', 'suite-fixture-graphing')
```

Comment:
Could the old implementation have been kept by re-ordering items in _suite-fixture-graphing.xml_?

Reply:
Yes, but we would also have had to update the suite generate scripts by adding a ```sorted``` to make the graph configuration deterministic.  I prefer to make the tests more flexible where possible.

Reply:
> we would also have had to update the suite generate scripts by adding a `sorted` to make the graph configuration deterministic

I'm not sure this is true. All dicts in Python 3.6+ now behave the same as `OrderedDict`. That is, they preserve insertion order. In Python 3.7 this became a documented feature of the language spec. So as long as items are added to dicts in a deterministic order the iteration order should always be the same.

Aside: I think since dict ordering has become part of the language spec in 3.7, we can also depend on it in 3.6 since we are never going to downgrade to 3.5.

Reply:
However, as long as we need to support Python 2.x we will have to enforce ordering in some way. Bummer. I guess we can add removing `sorted` to the list of things we can do once we're off of Python 2.